### ISSUE TITLE
Incorrect focus order in bar charts keyboard navigation

### ISSUE THREAD

**Issue by PetarNikValchev** [highcharts/highcharts#17048]
#### Expected behaviour
If we have the following actions:
1. Navigate with the keyboard to the series in a bar chart (with `tab` key).
2. Change the focus of the bars in the series (with `arrow` keys) to any bar that is not the first one.
3. Navigate out of the series element to the next element (`tab` key) or to the previous element (`shift + tab`).
4. Navigate back to the series element (`tab` or `shift + tab` depending on if you navigated to the previous or to the next element).

I expect that when I navigate back to the series element, the focus will fall on the last bar that it was on before I navigated out of the series element. Or at least there should be an option to allow such behavior. 

#### Actual behaviour
When I navigate back to the series element, the focus always falls on the first bar instead of on the last bar that it was on before I navigated out of the series element.

#### Live demo with steps to reproduce
https://jsfiddle.net/mxw4ukez/1
Follow the steps I listed in **Expected behaviour**.

#### Product version
v9.3.3

#### Affected browser(s)
I tested on the following browser versions:
- Chrome Version 98.0.4758.102 (Official Build) (64-bit)
- Edge Version 98.0.1108.43 (Official Build) (64-bit)

**Comment by oysteinmoseng** [comment]
Hi, marking this as a feature request since it currently works as expected from our side. Our user testing has indicated that the current behavior is what users expected, and they were sometimes confused if the chart remembered the selection from earlier.

With other web components the common pattern is also usually not to remember arrow key focus, for example in menus or tab controls with manual activation.

We could however introduce an option to change the behavior to operate as you describe.

**Comment by PetarNikValchev** [comment]
Hi @oysteinmoseng! Having an option to change the behavior would be great. Thanks!

**Comment by PetarNikValchev** [comment]
Hi @oysteinmoseng,

When can we expect this option to be released?

**Comment by oysteinmoseng** [comment]
Hi @PetarNikValchev, it should be relatively easy to implement, I can prioritize looking at it early next week

**Comment by PetarNikValchev** [comment]
Hi @oysteinmoseng, I can see this issue was fixed. Do you know when we can expect the fix to be released?

**Comment by oysteinmoseng** [comment]
@PetarNikValchev Hi, we are aiming for a new official Highcharts release in the near future, most likely next week. Is this okay for you?

**Comment by PetarNikValchev** [comment]
Hi @oysteinmoseng, next week works for us. Thanks!
